package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lock",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
I'm curious as to what of this pull request merited a bump in versioning...unless updating the README and the eslint version was enough for a whole subversion bump.

Reply:
This is a new version that captures [the last 18 months of changes](https://github.com/mozilla-iam/auth0-custom-lock/blob/master/CHANGELOG.md#170---2020-02-14). So the version increment isn't just the contents of this PR, it's the items in the CHANGELOG that have been committed after 1.6.0 but haven't been wrapped in a version.

My hope is to cut a version so that the CHANGELOG is accurate and describes what's live.

Reply:
Gotchya. So although this PR doesn't contain all the changes that are under the subversion bump, this is an attempt to wrap up all those changes in a comprehensive changelog/version bump and move forward.